### PR TITLE
Handle backtickd names in glue extraction

### LINKED ISSUE #1619: glue symbol extraction should treat x and `` `x` `` the same
_Originally posted by @AshesITR in https://github.com/r-lib/lintr/pull/1612#discussion_r987609552_

```r
x <- 1
`x` <- 2
glue::glue("{x} is the same as {`x`}")
# 2 is the same as 2
```

### PR DESCRIPTION
**Pull request by MichaelChirico**
Closes #1619

### CODE REVIEW

File: R/zzz.R [hunk 1]
```diff
@@ -249,8 +249,8 @@ settings <- NULL
   toset <- !(names(op_lintr) %in% names(op))
   if (any(toset)) options(op_lintr[toset])
 
-  # This is just here to quiet R CMD check
-  if (FALSE) backports::import
+  # requires R>=3.6.0
+  backports::import(pkgname, "str2lang")
```
Comment (MichaelChirico) [inline R/zzz.R:253]: Is this not the proper way to use backports? It seems to have failed to import `str2lang` in our older tests...

Comment (MichaelChirico) [reply R/zzz.R:253]: Looks like a {backports} bug maybe? https://github.com/r-lib/backports/issues/68

**Comment by MichaelChirico** [comment]
Bump: @AshesITR. Ping if you don't have time & I can re-assign to Indrajeet, thanks!

**Comment by MichaelChirico** [comment]
@IndrajeetPatil could you PTAL here?

**Review by IndrajeetPatil — APPROVED** [review]
LGTM! Cool unit test. 😎

**Pull request merged**